### PR TITLE
DOCU-2369: Removes prompts from instruction commands

### PR DESCRIPTION
**Pull request by rspurgeon**
Additionally:
* attempts to standardize some of the layout of command -> expected results to
match what was started in https://github.com/Kong/docs.konghq.com/pull/4064
* Adds a CTA at the bottom of the guide
* updates command expected output to more modern responses

### Reason
Removes the command prompts from the instruction commands, allows a user to more easily copy and paste the instructions

### Testing
Locally built and viewed the changed page

### CODE REVIEW

**Comment by rspurgeon** [comment]
Incorrectly reference DOCU-2377, instead this is related to DOCU-2369

File: src/kubernetes-ingress-controller/guides/getting-started.md [hunk 1]
```diff
@@ -17,17 +17,18 @@ If you've not done so, please follow one of the
 If everything is setup correctly, making a request to Kong should return back
 a HTTP 404 Not Found.
```
Comment (cloudjumpercat) [inline src/kubernetes-ingress-controller/guides/getting-started.md:18]:
```suggestion
a HTTP `404 Not Found` status code.
```

File: src/kubernetes-ingress-controller/guides/getting-started.md [hunk 2]
```diff
@@ -306,3 +309,9 @@ HTTP requests with /foo -> Kong enforces rate-limit -> echo server
 HTTP requests with /bar -> Kong enforces rate-limit +   -> echo-server
    on example.com          injects my-request-id header
 ```
+
+## Next steps
+
+* To learn how to secure proxied routes, check out the [ACL and JWT Plugins Guide](/kubernetes-ingress-controller/{{page.kong_version}}/guides/configure-acl-plugin/)
```
Comment (cloudjumpercat) [inline src/kubernetes-ingress-controller/guides/getting-started.md:315]:
```suggestion
* To learn how to secure proxied routes, see the [ACL and JWT Plugins Guide](/kubernetes-ingress-controller/{{page.kong_version}}/guides/configure-acl-plugin/).
```

File: src/kubernetes-ingress-controller/guides/getting-started.md [hunk 3]
```diff
@@ -306,3 +309,9 @@ HTTP requests with /foo -> Kong enforces rate-limit -> echo server
...
+* The [External Services Guide](/kubernetes-ingress-controller/{{page.kong_version}}/guides/using-external-service/) explains how to proxy services outside of your Kubernetes cluster
```
Comment (cloudjumpercat) [inline src/kubernetes-ingress-controller/guides/getting-started.md:316]:
```suggestion
* The [External Services Guide](/kubernetes-ingress-controller/{{page.kong_version}}/guides/using-external-service/) explains how to proxy services outside of your Kubernetes cluster.
```

**Review by cloudjumpercat — CHANGES_REQUESTED** [review]

**Pull request merged**